e3db/types/query_params.py

```diff
@@ -0,0 +1,33 @@
+import uuid
+
+class Params():
+    def __init__(self, cond="OR", strat="EXACT", keys=None, values=None, writer_ids=None, user_ids=None, record_ids=None, content_types=None, plain={}):
+        # python3 is picky about uuids, cant pass a uuid to uuid.UUID
+        writer_ids = [str(i) for i in writer_ids]
+        record_ids = [str(i) for i in record_ids]
+        user_ids = [str(i) for i in user_ids]
+
+        self.__cond = cond
+        self.__strat = strat
+        self.__writer_ids = [uuid.UUID(i) for i in writer_ids]
```

Comment:
You can, also do `uuid.UUID(str(i))`

It does mean that ids can only be passed in as things that serialize to uuid form. I don't expect users to pass in a list of bytes that are a uuid but if they did it would break. This is probably a case where given the typing style of python adding a doc to specify the expected type of each param  would be helpful. 

I might argue that those values that get passed in should either be UUIDs or a value that can passed to the UUID constructor. and then have the method do something like `[i if isinstance(i, uuid.UUID) else uuid.UUID(i) for i in writer_ids]`